src/components/edit-modal.js

```diff
@@ -1,26 +1,28 @@
 import React, { Component } from 'react'
 import { connect } from 'react-redux'
 import { Form, Checkbox, Button, Modal, Icon } from 'semantic-ui-react'
```

Comment:
merge this edit-modal file into edit-file file

Reply:
In tabular-view, we need this edit-modal independently so its better to keep it separated.